src/shared/components/Header/Navigation/component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Navigation, ToggleMenu, SearchButton } from './styles';
+import { MenuListItemI } from '../interfaces';
+import SVGSearch from '../../../icons/Search';
+
+type NavigationProps = {
```

Comment:
let's call it `TNavigationProps`, to highlight it's a type

Reply:
eslint displays an error